docs/How-To/cli/README.md

```diff
@@ -32,15 +32,17 @@ In order to test out the following command-line interfaces, you need to set up a
 
 ### MsgCreateFixedAmountPlan
 
-This is one of the private plan type messages that anyone can create. A fixed amount plan plans to distribute amount of coins by a fixed amount defined in `EpochAmount`. Internally, `PrivatePlanFarmingPoolAddress` is generated and assigned to the plan and the creator should query the plan and send amount of coins to the farming pool address so that the plan distributes as intended. There is a fee `PlanCreationFee` paid upon plan creation to prevent from spamming attack.
+Anyone can create this private plan type message. A fixed amount plan plans to distribute amount of coins by a fixed amount defined in `EpochAmount`. Internally, `PrivatePlanFarmingPoolAddress` is generated and assigned to the plan. The creator queries the plan and sends amount of coins to the farming pool address so that the plan distributes as intended. To prevent spamming attacks, a `PlanCreationFee` fee must be paid on plan creation.
 
-Create a file name `private-fixed-plan.json`. This private fixed amount farming plan intends to provide 100ATOM per epoch (measured in day) relative to the rate amount of denoms defined in staking coin weights.
+Create a `private-fixed-plan.json` file. This private fixed amount farming plan intends to provide 100ATOM per epoch (measured in day), relative to the rate amount of denoms that is defined in staking coin weights.
 
-- `name`: is the name of the farming plan. It can be any name you prefer to be stored in a blockchain network. It allows duplicate value.
-- `staking_coin_weights`: is the distributing amount for each epoch. An amount should be decimal, not an integer. The sum of total weight must be 1.000000000000000000
-- `start_time`: is start time of the farming plan 
-- `end_time`: is end time of the farming plan
-- `epoch_amount`: is an amount that will be distributed per epoch as an incentive for staking denoms defined in the staking coin weights.
+- `name`: the name of the farming plan can be any name to store in a blockchain network, duplicate values are allowed
```

Comment:
can you generate this content from the code? like https://docs.starport.com/cli/

Apply edits made here to all parameter descriptions throughout, I didn't edit each section that was repetitive

Reply:
Yes, indeed. I see that Starport uses `Vue` to bootstrap the documentation to the site if you look at the [docs folder](https://github.com/tendermint/starport/tree/develop/docs). But this is something that needs to be discussed within the team. I will raise an issue to discuss.

Reply:
https://github.com/tendermint/farming/issues/213